app/controllers/articles_controller.rb

```diff
@@ -19,6 +21,8 @@ def destroy
   end
 
   def update
+    params[:article][:categories].shift
+    add_category_to_article
```

Comment:
Since you provide the ability to to add multiple categories to the article, you probably want to rename this method to reflect that. `add_categories_to_article`